shimmer/develop/what-is-shimmer/layer-1-landscape.md

```diff
@@ -0,0 +1,141 @@
+| [IOTA Smart Contract (ISC)](#iota-smart-contracts-isc) Support | No                                                                                                     | On a [private Tangle](https://wiki.iota.org/shimmer/smart-contracts/guide/development_tools/docker_preconfigured/)                                                                                                                                                                                                                                                                                                                                                                            |
+
+## Stability
+
+Shimmer is designed as a staging network to test features publicly before deploying them on the IOTA network, so it
+will always have more features than IOTA. As this may come at the cost of some stability, before any protocol changes are deployed on the [Shimmer Mainnet](https://wiki.iota.org/shimmer/develop/endpoints/shimmer/), they have been tested
+in the [Shimmer Testnet](https://wiki.iota.org/shimmer/develop/endpoints/testnet/).
+
+## Store Data
+
+### IOTA
+
+In IOTA, you can attach
+an [IndexationPayload](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#indexationpayload)
+to
+any [SignedTransaction](https://wiki.iota.org/iota.rs/explanations/messages_payloads_and_transactions/#signedtransaction).
+The IndexationPayload allows you to add an index to a data message and arbitrary data.
+
+### Shimmer
+
+You can use the [INX-PoI](https://wiki.iota.org/shimmer/inx-poi/welcome/) node extension to verify if any piece of data
+was included in the Tangle. This feature does not rely on the [Chronicle](https://wiki.iota.org/shimmer/chronicle/welcome/)
+permanode, so it drastically reduces the node operator's storage requirements.
+
+## Tokenization
+
+### IOTA
+
+The IOTA network does not support custom tokens yet.
+
+### Shimmer
+
+The Stardust protocol introduces
+a [tokenization framework](https://wiki.iota.org/shimmer/introduction/explanations/what_is_stardust/tokenization/)
+that enables [Alias outputs](https://wiki.iota.org/shimmer/introduction/explanations/ledger/alias/),
+[NFTs](https://wiki.iota.org/shimmer/introduction/explanations/ledger/nft/)
+and [native tokens and foundries](https://wiki.iota.org/shimmer/introduction/explanations/ledger/foundry/)
+on layer 1, as long as you can cover the
+minimum [storage deposit](#storage-deposits).
+
+## Storage Deposits
+
+### IOTA
+
+There are no data storage deposits in the IOTA network, as it relies on [dust protection](https://wiki.iota.org/introduction/reference/details/#dust-protection), which means you can only
+send microtransactions of less than 1Mi if the target address already has at least 1Mi.
```

Comment:
Do we want to mention the `SigLockedDustAllowanceOutput` here? If not, I would keep it more vague and say something like:
 ```suggestion
send microtransactions of less than 1Mi if the target address mets some special criteria.
```

and maybe link to the [TIP](https://wiki.iota.org/tips/tips/TIP-0015/) or something